Automattic/pocket-casts-android
# Display podcast sharing UI

## Description

This PR displays podcast reimagine sharing page. It is only UI and without view pager for card selection. Also, the issue with tablets described [here](https://github.com/Automattic/pocket-casts-android/pull/2518#pullrequestreview-2198387211) should be fixed now.

Designs: HR2vcQrWcS0scsolKZBitg-fi-2203_2359

## Testing Instructions

1. Share a podcast.
2. Verify the UI and the page's behavior.

Because it is the first integration of social platforms bar you can also test it. Check if the app detects correctly if the apps installed on your device. Note that the page doesn't update itself live for installed apps. You need to reopen the page to verify installations.

## Screenshots or Screencast 

| Portrait | Landscape |
| - | - |
| ![Screenshot_20240725-121040](https://github.com/user-attachments/assets/2e3545cf-f8a1-4950-a383-ee92adad59f8) | ![Screenshot_20240725-121825](https://github.com/user-attachments/assets/b668d2df-5220-41cc-9a35-0c5fe7de51cd) |

## Checklist
 
#### I have tested any UI changes...
